src/test/java/com/javabom/lotto/domain/ticket/LottoTicketDispenserTest.java

```diff
@@ -0,0 +1,51 @@
+package com.javabom.lotto.domain.ticket;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoTicketDispenserTest {
+
+    @DisplayName("로또 발급기가 원하는대로 로또 티켓을 발급하는지 확인")
+    @Test
+    void getAutoTickets() {
+        // given
+        LottoTicketDispenser lottoTicketDispenser = new LottoTicketDispenser(new FixedLottoNumberGenerator());
```

Comment:
```suggestion
        LottoTicketDispenser lottoTicketDispenser = new LottoTicketDispenser(() -> Arrays.asList(new LottoNumber(1),....));
```

이렇게 하는 거랑 `FixedLottoNumberGenerator`를 사용하는 거랑 어떤 차이가 있을까?

Reply:
뭐가 들어있는 지 알수 없게되서 테스트가 비직관적이게된다?
파라미터들이 많아져도 따로 빼는것보단 낫구나
엄청나게 길어지는게 아닌 이상 그냥 넣는게 나으려나